CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 ### Changed
 - Use INFORMATION_SCHEMA system table in mysql2 adapter to count rows
 
+## 0.8.0
```

Comment:
FYI

> Keep an `Unreleased` section at the top to track upcoming changes.

cf. https://keepachangelog.com/en/1.0.0/

Reply:
I mean 0.8.0 includes "Use INFORMATION_SCHEMA system table in mysql2 adapter to count rows".

Reply:
Oh sorry, I did not understand how to write CHANGELOG. thanks.
`Unreleased` section was the top, over the `Changed`, I just added "Use INFORMATION_SCHEMA ~" line to new version section.

✅  Fixed https://github.com/hogelog/dmemo/pull/119/commits/ca6a964076ed0d31580ec0ddaf464f1054bcdefd